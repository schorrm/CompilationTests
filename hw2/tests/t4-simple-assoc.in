int bar() {
    if (bar <= bar >= bar) {
        print("Then you got your associativity wrong");
    }
}
