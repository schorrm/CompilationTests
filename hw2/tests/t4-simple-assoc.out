31: Type -> INT
5: RetType -> Type
7: Formals -> epsilon
36: Exp -> ID
36: Exp -> ID
line 2: syntax error
